README.md

```diff
@@ -1 +1,179 @@
+
+- **Add category**
+- **Add transaction**
+- **Authenticate user**
+- **Authorize user**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://budget-app-oa5o.onrender.com/)
+
+- [Presentation Video Link](https://www.loom.com/share/1220e448728243fa9de0bb2d4483f783?sid=babae4f9-8b89-41b9-8718-63aeb8188482)
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Example command:
+
+```sh
+ gem install rails
+```
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone git@github.com:CaptainBawa/Budget-App.git
+```
+### Install
+
+Install this project with:
+
+```sh
+  cd Budget-App
+  bundle install
+```
+
+```sh
+  rails db:create
+```
+
+```sh
+  rails db:migrate
+```
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  rails rspec spec/model/category_test.rb
+```
+### Deployment
+
+You can deploy this project using:
+
+```sh
+  https://render.com/
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Collins Bawa**
+
+- GitHub: [@CaptainBawa](https://github.com/CaptainBawa)
+- Twitter: [@BawaCollins](https://twitter.com/BawaCollins)
+- LinkedIn: [@captainbawa](https://www.linkedin.com/in/captainbawa/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Add complex styles**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/CaptainBawa/Budget-App/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+Dear Viewers!
+
+I'm excited to be working on this project and I would be grateful for your support! By supporting this project, you are not only helping me, but you are also contributing to something meaningful that can make a difference. Your support will give me the motivation and resources necessary to keep moving forward with this project and to ensure its success. So, whether it's through your kind words, by giving a star if you like this project, your financial support, or simply by spreading the word, your support means everything to me. Thank you in advance for your support, and I can't wait to share with you the amazing things we will achieve together.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the Microverse full-stack curriculum for the inspiration and guidance.
+
```

Comment:
- [ ] It's a nice and smart thing to give appropriate credit to **[Gregoire Vella on Behance](https://www.behance.net/gregoirevella)**, the author of [the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). Kindly ensure to add this under the **Acknowledgements** section of your ReadMe.